lib/BillTechLinkApiService.php

```diff
@@ -110,7 +110,7 @@ public static function getLinkData($linkRequest)
 			if (!$linkData) {
 				throw new Exception("Could not fetch link data by document id: " . $linkRequest->srcDocumentId);
 			}
-			$linkData['title'] = self::getDocumentTitle($linkData['fullnumber'], $linkData['comment'], $linkData['name']);
+			$linkData['title'] = self::getDocumentTitle($linkData['fullnumber'], $linkData['comment'], $linkData['fullname']);
```

Comment:
O co chodzi?

Reply:
Planowo w przypadku braku fullnumber i comment na encji document tutył płatności ma być "Faktura Imię Nazwisko". Niestety c.name to tylko imię, więc albo powinienem wyciągać c.name . ' ' . c.lastname albo właśnie d.name, które jest imieniem i nazwiskiem razem. fullname dlatego, że kolumna name już istnieje i nadałem alias.

Reply:
Ok, nie wiedziałem, że d.name zawiera imię i nazwisko, bo ostatnio chyba ustaliliśmy, że samo imię. Skoro tak jest jak wyżej piszesz to ma sens, żeby było 'fullname' 👍